string action
int32 position
int32 speed
int32 force
bool forceDetection
float32 incrementXaxis
float32 incrementYaxis
float32 incrementZaxis
float32[6] robotJoints
geometry_msgs/Pose robotPose
